src/main/resources/db/changelog/0003-create-table-financial_transaction_categories.yaml

```diff
@@ -1,6 +1,6 @@
 databaseChangeLog:
   - changeSet:
-      id: "7"
+      id: "create table financial_transaction_categories"
```

Comment:
Table name "financial_transaction_categories" contains an underscore characters. 
But in other file:
`src/main/resources/db/changelog/0002-create-table-financial_transactions.yaml`
"create table financial transactions"
doesn't.

I think it should be the same convention.